swap_server_client.go

```diff
@@ -771,6 +772,55 @@ func (s *grpcSwapServerClient) MuSig2SignSweep(ctx context.Context,
 	return res.Nonce, res.PartialSignature, nil
 }
 
+// MultiMuSig2SignSweep calls the server to cooperatively sign an input in
+// a batch transaction that attempts to sweep multiple htlcs at once. This
+// method is called once per input signed. The prevoutMap is a map of all the
+// prevout information for each spend outpoint. Returns the server's nonce and
+// partial signature.
+func (s *grpcSwapServerClient) MultiMuSig2SignSweep(ctx context.Context,
```

Comment:
nit: could use a functional options pattern or a config here to help readability.

Reply:
https://github.com/lightninglabs/loop/pull/634#discussion_r1339139323